src/pages/design/icons/index.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Icon Overview
+---
+
+import { graphql } from 'gatsby';
+
+import IconShowcase from '../../../examples/design/icons/IconShowcase.tsx';
+import IconShowcaseCode from '!!raw-loader!../../../examples/design/icons/IconShowcase.tsx';
+
+<CodeExample code={IconShowcaseCode}>
+  <IconShowcase />
+</CodeExample>
+
+### Usage
+
+Garden’s icon library is implemented using SVG. Icons come two sizes: 12px and 16px.
+The 16px icons can also be doubled to create 32px icons.
+
+#### Styles
+
+import IconStyles from '../../../examples/design/icons/IconStyles.tsx';
+import IconStylesCode from '!!raw-loader!../../../examples/design/icons/IconStyles.tsx';
+
+<CodeExample code={IconStylesCode}>
+  <IconStyles />
+</CodeExample>
+
+Icons also come in two styles: outline and filled.
+While outlined icons are the default icons to use, filled icons are used to show selected states.
+
+### 12px and 16px
+
+The default size is 16px, and should be used for the majority of cases.
+In certain instances where 16px icons will break the line height or will not fit in a small space, 12px icons should be used.
+
+<BestPractice>
+  <Do imageSource={props.data.defaultIconSize.childFile.childImageSharp.gatsbyImageData}>
+    <p>16px is the default icon size. Use 16px icons where possible.</p>
+  </Do>
+</BestPractice>
+
+<BestPractice>
+  <Do imageSource={props.data.smallerIconSize.childFile.childImageSharp.gatsbyImageData}>
+    <p>Use 12px icons for compact spaces where 16px icons can’t fit.</p>
+  </Do>
+</BestPractice>
+
+### 32px
+
+32px icons are the scaled-up versions 16px icons. They are used for illustration only.
+
+<BestPractice>
+  <Do imageSource={props.data.illustrationIconSize.childFile.childImageSharp.gatsbyImageData}>
+    <p>Use 32px icons for illustration.</p>
+  </Do>
+</BestPractice>
+
+<BestPractice>
+  <Dont
+    imageBackgroundColor="#D2D5D8"
+    imageSource={props.data.uiIconSize.childFile.childImageSharp.gatsbyImageData}
+  >
+    <p>Don’t use 32px icons for UI.</p>
```

Comment:
@bethanybeech Don't recall if I saw this content in the last iteration we worked on together but regardless, can we change this to something like this "Don't use 32px icons for functions in the user interface." I want to be more clear here since illustrations can technically be in the UI. What are your thoughts Bethany?